onnx_xla/translations/translate_unsqueeze.cc

```diff
@@ -0,0 +1,42 @@
+#include "onnx_xla/operator_registry.h"
+
+namespace onnx_xla {
+// Translate Unsqueeze by using XLA's Reshape
+onnxStatus translateUnsqueeze(const Node& n,
+                              XlaBuilder& builder,
+                              ValueOpMap& valueToOp) {
+  // Set origShape and axes
+  std::vector<int64> origShape = OperatorRegistry::parseOnnxInputSizes(n, 0);
+
+  if (!n.hasAttribute(kaxes)) {  // TODO ENFORCE
```

Comment:
let's prioritize adding this ENFORCE :-)

Reply:
yup :) will do after operator PR's go through